SecondHomework/Logistic/ParcelManager.cs

```diff
@@ -49,4 +58,21 @@ public Result RemoveParcel(Guid id, List<Parcel> parcelsList)
         }
         return result;
     }
+
+    public List<List<Parcel>> GetParcelsByWeight()
+    {
+        var upTo1Kg = _parcels.Where(parcel => parcel.Weight <= 1).ToList();
+        var upTo2Kg = _parcels.Where(parcel => parcel.Weight > 1 && parcel.Weight <= 2).ToList();
+        var upTo5Kg = _parcels.Where(parcel => parcel.Weight > 2 && parcel.Weight <= 5).ToList();
+        var upTo10Kg = _parcels.Where(parcel => parcel.Weight > 5 && parcel.Weight <= 10).ToList();
+        var upTo20Kg = _parcels.Where(parcel => parcel.Weight > 10 && parcel.Weight <= 20).ToList();
+        var moreThan20Kg = _parcels.Where(parcel => parcel.Weight > 20).ToList();
+
+        return new List<List<Parcel>> { upTo1Kg, upTo2Kg, upTo5Kg, upTo10Kg, upTo20Kg, moreThan20Kg };
```

Comment:
See comment about using enum here and return `Dictionary<EnumWeightCategories, List<Parcel>>`